jax/core.py

```diff
@@ -2316,27 +2426,41 @@ def check_eqn(prim, in_avals, params):
     out_avals = [out_avals]
   return out_avals, effects
 
-def check_call(ctx_factory, prim, in_avals, params):
+def _check_call(ctx_factory, prim, in_atoms, params):
```

Comment:
Might be a nit, but why did this pick up a leading underscore?

Reply:
I wanted to make sure there were no existing callers so I changed the name, and also if the function is just a helper not to be used outside this module having an underscore is a nice hint. WDYT?

Reply:
No strong opinion either way – just making sure I wasn't missing some new meaning. In this case, I'd update `check_map` to `_check_map` as well to be consistent.

Reply:
By the way, if our current `core` were under `_src` (maybe with select symbol exports via `jax.core`), I would slightly prefer no leading underscores.

Reply:
Done.